client/me/purchases/manage-purchase/index.tsx

```diff
@@ -639,6 +642,76 @@ class ManagePurchase extends Component<
 		return null;
 	}
 
+	renderRefundText() {
+		const { purchase, translate } = this.props;
+
+		// Hide if refund window has lapsed.
+		if ( ! purchase?.isRefundable || ! purchase?.refundText || ! purchase?.mostRecentRenewDate ) {
+			return;
+		}
+
+		const ONE_DAY_IN_MILLISECONDS = 24 * 60 * 60 * 1000;
+		const lastRenewalDate = new Date( purchase?.mostRecentRenewDate );
```

Comment:
Super minor but we don't need the optional chaining here because there's a guard for `purchase` above so it won't ever be falsy here. It might be wise in fact to remove it because `new Date(undefined)` produces an invalid date object and if the guard above were ever removed, this could cause hidden bugs.

```suggestion
		const lastRenewalDate = new Date( purchase.mostRecentRenewDate );
```

Reply:
Ah, thanks so much @sirbrillig! Those confused me, so I appreciate the extra explanation too. 

All fixed now. :)